src/main/java/org/passay/RuleResultDetail.java

```diff
@@ -71,6 +105,6 @@ public String getErrorCode()
   @Override
   public String toString()
   {
-    return String.format("%s:%s", errorCode, parameters);
+    return String.format("%s:%s", Arrays.asList(errorCodes), parameters);
```

Comment:
`Arrays#toString` is probably better suited.

Reply:
Fixed.